usr/lib/ica_s390_stdll/ica_specific.c

```diff
@@ -4846,3 +4914,60 @@ CK_RV token_specific_ecdh_pkcs_derive(STDLL_TokData_t *tokdata,
 }
 
 #endif
+
+CK_RV token_specific_object_add(STDLL_TokData_t *tokdata, SESSION *sess,
+                                OBJECT *obj)
+{
+    ica_private_data_t *ica_data = (ica_private_data_t *)tokdata->private_data;
+    CK_OBJECT_CLASS class;
+    CK_KEY_TYPE keytype;
+#ifndef NO_EC
+    ICA_EC_KEY *eckey = NULL;
+    unsigned int privlen;
+    EVP_PKEY *ec_key = NULL;
```

Comment:
I really don't like this naming.  why not `ica_eckey` and `ossl_eckey`?  Sure, it is more typing, but it better captures what the variables are about (and not only differs in a meaningless underscore).

Reply:
you are right. Changed it to the suggested names.